lib/player.js

```diff
@@ -3162,6 +3162,16 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
 
     goog.asserts.assert(typeof(config) == 'object', 'Should be an object!');
 
+    // Deprecate 'streaming.forceTransmuxTS' configuration.
```

Comment:
Please add something about this to docs/tutorials/upgrade.md.

Reply:
Done!